plugins/versionpress/src/Utils/RequirementsChecker.php

```diff
@@ -121,6 +120,16 @@ public function __construct($database, DbSchemaInfo $schema, $checkScope = Requi
             'help' => 'VersionPress automatically tries to secure certain locations, like `wp-content/vpdb`. You either don\'t have a supported web server or rules cannot be enforced. [Learn more](http://docs.versionpress.net/en/getting-started/installation-uninstallation#supported-web-servers).'
         ];
         if ($checkScope === RequirementsChecker::SITE) {
+            if (file_exists(VP_PROJECT_ROOT . '/composer.json')) {
+                $this->requirements[] = [
+                    'name' => 'Composer scripts',
+                    'level' => 'warning',
+                    'fulfilled' => $this->testComposerJson(),
+                    // @codingStandardsIgnoreLine
+                    'help' => 'Your project uses Composer but `pre-update-cmd` or `post-update-cmd` scripts in `composer.json` are already defined. VersionPress needs to run a WP-CLI commands in these scripts to automatically commit changes of Composer packages. Please update your scripts to run [the WP-CLI commands](http://docs.versionpress.net/en/feature-focus/composer).'
```

Comment:
Please create issue in documentation to create content for `feature-focus/composer`.